draft-ietf-acme-acme.md

```diff
@@ -395,8 +402,8 @@ An ACME server MUST implement the "ES256" signature algorithm {{!RFC7518}} and S
 The "jwk" and "kid" fields are mutually exclusive. Servers MUST reject requests
 that contain both.
 
-For newAccount requests, and for revokeCert requests authenticated by certificate
-key, there MUST be a "jwk" field. This field MUST contain the public key
+For newAccount requests, and for revokeCert requests authenticated by certified
+keys, there MUST be a "jwk" field. This field MUST contain the public key
```

Comment:
I don't think changing "certificate key" to "certified keys" makes sense in this context. As a concrete example the account key that issued the certificate specified in the revocation request would be a certified key (in the sense it is authorized to do the revocation), but it wouldn't be a revocation request that MUST have a "jwk" field (it would carry the mutually exclusive "kid" field). 

This is specifically talking about the case where a request to revoke an issued end-entity certificate is authenticated using a JWK corresponding to the to-be-revoked certificate's public key.

Reply:
Ah so maybe it's just missing "a" in the sentence "and for revokeCert requests authenticated by a certificate key".

Reply:
@seanturner :+1: I think adding a "a" in the sentence does clarify. Good suggestion

Reply:
@seanturner Ping on this?